app/src/main/java/mozilla/lockbox/model/ToastNotificationViewModel.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+data class ToastNotificationViewModel(
+    val text: String? = null,
+    @StringRes val strId: Int? = null,
```

Comment:
Make this non-optional. The text should always be inserted into a translated string.